src/elasticodbc/es_info.cpp

```diff
@@ -83,7 +83,7 @@ const std::unordered_map< int, std::vector< int > > sql_es_type_map = {
       ES_TYPE_UNSUPPORTED}},
     {SQL_TYPE_TIMESTAMP, {ES_TYPE_DATETIME}}};
 
-// TODO AE-152: Refactor this to improve performance
+// TODO #325 (SQL Plugin)- Refactor this to improve performance
```

Comment:
I think this was referring to refactoring the way the polymorphism is setup and function calls are performed, as is independent of the SQL Plugin. I don't think this is a performance bottleneck, so perhaps it's not required, but I don't think 325 is relevant to this.

Reply:
removed